src/Microsoft.SqlTools.ServiceLayer/DacFx/DacFxService.cs

```diff
@@ -150,6 +151,45 @@ public async Task HandleDeployRequest(DeployParams parameters, RequestContext<Da
             }
         }
 
+        /// <summary>
+        /// Handles request to generate deploy script
+        /// </summary>
+        /// <returns></returns>
+        public async Task HandleGenerateDeployScriptRequest(GenerateDeployScriptParams parameters, RequestContext<DacFxResult> requestContext)
+        {
+            try
+            {
+                ConnectionInfo connInfo;
+                ConnectionServiceInstance.TryFindConnection(
+                        parameters.OwnerUri,
+                        out connInfo);
+                if (connInfo != null)
+                {
+                    SqlConnection sqlConn = ConnectionService.OpenSqlConnection(connInfo, "GenerateScript");
```

Comment:
I looked into the implementation of the Dac operations, it seems to me we don't really need a sqlconnection object, we only need the connection string to be passed to the DacServices object. Can we remove all the OpenSqlConnection call in this file?

Reply:
Ok I removed all the OpenSqlConnection calls and build the connection string instead